crates/aptos-rosetta/src/client.rs

```diff
@@ -528,13 +528,7 @@ impl RosettaClient {
         let mut keys = HashMap::new();
         keys.insert(delegator, private_key);
 
-        let operations = vec![Operation::add_delegated_stake(
-            0,
-            None,
-            delegator,
-            AccountIdentifier::base_account(pool_address),
-            amount,
-        )];
+        let operations = vec![];
```

Comment:
When this vector is empty, no operation is performed. Doesn't this mean that the test is practically removed?

Reply:
Okay, Actually we need to remove entire test for **delegation pool** module from `aptos-rosetta`. 
I made the changes